stdlib/LinearAlgebra/src/qr.jl

```diff
@@ -522,6 +522,10 @@ end
 
 ## Multiplication by Q
 ### QB
+QRCompactWYOrPackedQ{T,S} = Union{QRCompactWYQ{T,S}, QRPackedQ{T,S}}
+mul!(C::StridedVecOrMat{T}, AOrAdjA::Union{Adjoint{<:Any,<:QRCompactWYOrPackedQ{T,S}}, QRCompactWYOrPackedQ{T,S}}, B::StridedVecOrMat{T}) where {T, S} =
+    lmul!(AOrAdjA, copy!(C, B))
```

Comment:
Shouldn't it be `copyto!`?

Reply:
For the purpose of this function, they work the same

Reply:
Strange. Back then, I couldn't find a `copy` method instance that would work for matrices... Oh, I just noticed this has been introduced in Julia 1.1. Nevermind!